metricflow-semantics/metricflow_semantics/model/semantics/linkable_spec_index_builder.py

```diff
@@ -656,36 +636,35 @@ def _get_joined_elements_without_cache(
 
         return LinkableElementSet.merge_by_path_key((single_hop_elements, multi_hop_elements))
 
+    def _get_linkable_elements_accessible_from_semantic_model(
```

Comment:
Didn't mean to include this helper function in this commit but it's just a rearrangement of code and needed up the stack anyway